### DISCUSSION TITLE
Copy/Export/Import Apps

### DISCUSSION THREAD

**Discussion by AndrewIammatteo** [btcpayserver/btcpayserver#3439] [Ideas / Feature Requests]
Hi,

I have not come across the feature of being able to copy/export/import apps. This would be a very useful features. This has mostly been a headache for testing, but I am also working on a workaround for tips going to a separate Store wallet than the original purchase and this would be very helpful. (I am redirecting the base store app to the employee tipping apps and there needs to be one per employee)

Thanks

**Comment by Kukks** [comment]
We'll implement an api for apps. This should make it much easier to replicate apps.
